docs/book/quick-start.md

```diff
@@ -84,29 +84,82 @@ If you are at all uncertain about the source of a variable in your view, escape
 
 Routes determine which controller to call based on the URI and other information from the request.
 
-Configure a route and a controller in `module/Application/config/module.config.php`:
+A couple of array values are declared for routes and controllers are added to  `module/Application/config/module.config.php` as shown below:
 
 ```php
+<?php
+
+declare(strict_types=1);
+
+// Declaring our module's namespace here.  Allows us to automatically reference classes in the 
+// src folder of our module with pathnames relative to the src folder. 
```

Comment:
I don't think that explanation of namespaces (core PHP functionality) belongs here.

```suggestion
```

Reply:
> …that explanation of namespaces (core PHP functionality) belongs here.

Right, because this is a beginner's guide for laminas-mvc and not for PHP and OOP.